bower.json

```diff
@@ -1,26 +1,23 @@
 {
   "name": "adapt-vimeo",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/adaptlearning/adapt-vimeo.git"
-  },
   "version": "3.1.1",
   "framework": ">=5.19.1",
   "homepage": "https://github.com/adaptlearning/adapt-vimeo",
   "issues": "https://github.com/adaptlearning/adapt-vimeo/issues/new",
```

Comment:
Can `/new` be removed from this to mirror other plugins please

Reply:
Done